### ISSUE TITLE
Unable to drag and drop from Gear list into list

### ISSUE THREAD

**Issue by MatanyaLoewenthal** [galenmaly/lighterpack#127]
When I try to drag an item from my Gear list, the small dotted rectangle does not show up for most items. Once in a while, It will work, but always with an item such as an empty element. Same issues exist on the old version. Using latest chrome 74.0.3729.131 (Official Build) (64-bit), on Ubuntu 18.04.

**Comment by galenmaly** [comment]
Interesting! 

A couple things could be happening here:

1. The item is already in the current list. There's currently a limitation that you can only have an item in a list once, so the dotted rectangle won't show up for those items. If this is what's happening then let's mark this issue as a "confusing UX". Maybe could be solved by adding a tooltip on these items
2. The item isn't in the current list. This would be a bug that we should prioritize fixing.

Can you confirm which one it is?

Thanks!

**Comment by MatanyaLoewenthal** [comment]
Yeah, this seems to fall under confusing UX.

**Comment by jonjahr** [comment]
I can't drag an item from my gear list either.  The item isn't in the current list.  I'm using Chrome 75.0 on MacOS 10.14.4.  I googled a bit and found at least [one other person](https://backpackinglight.com/forums/topic/lighterpack-com-functionality-questions/) having trouble too.

Steps to reproduce:
- Create new blank list.
- Create one category in the list.
- Drag any piece of gear from the left column onto the new category.

Observed: On mousedown, the item drags with the mouse.  On hover, the category under the mouse turns yellow.  On mouseup, the item flies back to the left column, and doesn't get added to the new category.  (See screen recording GIF)

Happy to provide more info if you have trouble reproducing.

![lighterpack-drag2](https://user-images.githubusercontent.com/6802815/59885608-a4157280-9370-11e9-83f4-7e684342bdb3.gif)

**Comment by Lanjelin** [comment]
Try dragging it a little further, category should turn green before mouseup.

**Comment by jonjahr** [comment]
@Lanjelin Yes that worked, thank you!  (I dragged the item further right, almost to the Description column, the category turned green, and I could add it to the category.)  

The yellow background really fooled me into thinking that I had dragged it far enough into the drop zone.
